### ISSUE TITLE
Is there any benefit on datasets (small object detection task) with known labels?

### ISSUE THREAD

**Issue by dantetemplar** [lightly-ai/lightly#1817]
Do I get benefits when:
1. Pretrain with YOLO12 on images with lightly_train
2. Finetune trained model on the very same data but with common pipeline

I have dataset similar to this:
![Image](https://github.com/user-attachments/assets/981d091f-8c44-4b5e-9527-4c20d30f0944)
https://datasetninja.com/lacmus-drone-dataset

60k-80k number of images, up to 4k resolution.

**Comment by guarin** [comment]
Hi and thanks for your interest in LightlyTrain!

Given that your data is already fully labeled I don't think you'll get major improvements by pretraining first.

Btw, the LightlyTrain repo is over here: https://github.com/lightly-ai/lightly-train 😉

**Comment by dantetemplar** [comment]
Sorry for duplicating issue, thanks for the answer!
